### PR TITLE
README: clarify licenses

### PR DESCRIPTION
**Pull request by roman-khimov**
`rpc` can't use GPLv3, it's intended to be used by everyone and it's mostly autogenerated. @AnnaShaleva, @carpawell, @cthulhu-rider, please approve.

### CODE REVIEW

**Review by carpawell — DISMISSED** [review]

File: LICENSE.md [hunk 1]
```diff
@@ -1,675 +1,14 @@
...
-the GNU GPL, see <https://www.gnu.org/licenses/>.
-
-The GNU General Public License does not permit incorporating your
-program into proprietary programs. If your program is a subroutine
-library, you may consider it more useful to permit linking proprietary
-applications with the library. If this is what you want to do, use the
-GNU Lesser General Public License instead of this License. But first,
-please read <https://www.gnu.org/licenses/why-not-lgpl.html>.
+This repository provides several different components that use different
+licenses. Contracts, tests and commands use GPLv3 (see LICENSE.GPLv3.md),
```
Comment (AnnaShaleva) [inline LICENSE.md:2]: Link to the doc may be added.

File: LICENSE.md [hunk 2]
```diff
@@ -1,675 +1,14 @@
...
+while RPC bindings (95% of which are autogenerated code) and other
+code intended to be used as a library to integrate with these contracts
+is distributed under Apache 2.0 license (see LICENSE.Apache2).
```
Comment (AnnaShaleva) [inline LICENSE.md:5]: And here.

**Review by AnnaShaleva — APPROVED** [review]

File: LICENSE.md (hunk 1 above)
Comment (roman-khimov) [reply LICENSE.md]: Sure, it's markdown.

File: LICENSE.md (hunk 2 above)
Comment (roman-khimov) [reply LICENSE.md]: Done.

**Review by AnnaShaleva — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**